src/Git/Git.php

```diff
@@ -81,7 +81,7 @@ public function clone_no_checkout($repository)
 			return false;
 		}
 
-		$this->run_command("mv {$tmpdir}/.git {$this->repo_path}");
+		$this->run_command(['mv', $tmpdir . '/.git', $this->repo_path]);
```

Comment:
Non-blocking comment: there's a typo in the commit message here `s/no/not/g`

Reply:
Good spot! Fixed in a rebase I'll push post-approval.